lib/features/judge/data/local/judge_repository_imp.dart

```diff
@@ -39,4 +39,37 @@ class JudgeRepositoryImp extends JudgeRepository {
 
     return status;
   }
+
+  @override
+  Future<List<JudgeEntity>> getJudges() async {
+    final User? user = auth.currentUser;
+    final CollectionReference collectionRef;
+    final QuerySnapshot querySnapshot;
+    final List<JudgeEntity> judgesList = [];
+
+    if (user != null) {
+      collectionRef = db.collection(judges);
+      querySnapshot = await collectionRef.get();
+
+      for (var element in querySnapshot.docs) {
+        Map<String, dynamic> data = element.data() as Map<String, dynamic>;
+
+        List<dynamic> list = data["socialNetwork"];
+        List<String> socialNetwork =
+            list.map((e) => element.toString()).toList();
+
+        JudgeEntity judge = JudgeEntity(
+            id: data["id"],
+            name: data["name"],
+            about: data["about"],
+            socialNetwork: socialNetwork);
+
+        print("Rresultados: $judge");
+
+        judgesList.add(judge);
```

Comment:
because we are using freezed you can user the JudgeEntity.fromJson(data).
with that you avoid the manual parsing1

Reply:
the judges has photos so could be cool to show the judge photo in teh tile view

Reply:
Done